nilearn/decoding/searchlight.py

```diff
@@ -353,7 +358,48 @@ def fit(self, imgs, y, groups=None):
             self.n_jobs,
             self.verbose,
         )
+        self.process_mask_ = process_mask
+        self.masked_scores_ = scores
```

Comment:
Can you update the tests to check that those attributes are exist after fitting?